unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/Views/SearchBarView/SortDropdownView.cs

```diff
@@ -0,0 +1,98 @@
+﻿using DCL.Helpers;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using Object = UnityEngine.Object;
+
+internal class SortDropdownView : MonoBehaviour, IDeselectHandler
+{
+    public event Action<string> OnSortTypeSelected;
+
+    [SerializeField] private Transform buttonsContainer;
+    [SerializeField] private SortDropdownButton dropdownButtonBase;
+    [SerializeField] private ShowHideAnimator showHideAnimator;
+
+    private readonly Queue<SortDropdownButton> buttonsPool = new Queue<SortDropdownButton>();
+    internal readonly List<SortDropdownButton> activeButtons = new List<SortDropdownButton>();
+
+    private void Awake()
+    {
+        dropdownButtonBase.gameObject.SetActive(false);
+        dropdownButtonBase.OnSelected += OnSortButtonPressed;
+        buttonsPool.Enqueue(dropdownButtonBase);
+
+        gameObject.SetActive(false);
+    }
+
+    public int GetSortTypesCount()
+    {
+        return activeButtons.Count;
+    }
+
+    public void Show()
+    {
+        if (!gameObject.activeSelf)
+        {
+            gameObject.SetActive(true);
+        }
+
+        if (!(EventSystem.current is null))
```

Comment:
Maybe this line could be clearer in this way?

```suggestion
        if (EventSystem.current is not null)
```

Reply:
That's not supported by the c# versions < 9 :(
I changed it for `!= null`